samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+
+# Enable Direction Finding TX Feature including AoA and AoD
+CONFIG_BT_CTLR_DF=y
+CONFIG_BT_CTLR_DF_ANT_SWITCH_RX=n
```

Comment:
Not sure if important, but it's missing an empty newline at the end

Reply:
New line was added